SDL/PixelTriplet.h

```diff
@@ -2577,7 +2575,7 @@ namespace SDL {
 
     rzChiSquared = computePT5RZChiSquared(acc, modulesInGPU, lowerModuleIndices, rtPix, zPix, rts, zs);
 
-    if (pixelRadius < 5.0f * kR1GeVf) {
+    if (/*pixelRadius*/ 0 < 5.0f * kR1GeVf) {  // FIXME: pixelRadius is not defined yet
```

Comment:
@YonsiG this is the part I was talking about during the meeting

Reply:
I wonder whether this was contributing to #265. I'll check this, as well as fix the other non-deterministic kernels that @YonsiG found.

Reply:
Thank you for finding it Slava! I'm just curious, why do we want to only use the rzChiSquared only for < 5GeV? For larger pt, in principal, the track on rz should be very straight line

Reply:
good point, in rz computations hopefully do not explode numerically and we could safely apply the cut everywhere (well, we do already, by virtue of the bug here).

Note though that `computePT5RZChiSquared` is not using the helix correction.
Why? Doesn't this lead to inefficiency at low pt?

Reply:
Hi Slava, thank you for your explanation! I don't remember there is a reason to keep the linear constraint for pT5 r-z. I think it's just we haven't applied the helix here yet. Can add it on my to-do list if necessary.